casbah/src/main/scala/akka/contrib/persistence/mongodb/CasbahPersistenceReadJournaller.scala

```diff
@@ -62,7 +67,7 @@ class CurrentAllEvents(val offset: Long, val driver: CasbahMongoDriver) extends
 
   override protected def initialCursor: Stream[Event] = {
 
-    val query = MongoDBObject(TIMESTAMP->MongoDBObject("$gte"->offset))
+    val query = MongoDBObject(TIMESTAMP->MongoDBObject("$gt"->offset))
```

Comment:
I have fixed the checkpoint in our side, as discussed it should remain as "$gte"